src/constants.ts

```diff
@@ -161,3 +161,12 @@ var guidedTours = {
 };
 
 _.set(window, 'OPENSHIFT_CONSTANTS.GUIDED_TOURS', guidedTours);
+
+var publisherSynonyms = {
+  'Vendor B': 'Vendor B, Inc.',
+  'Vendor B, Inc': 'Vendor B, Inc.',
+  'Vendor B, Co.': 'Vendor B, Inc.'
+};
```

Comment:
Let's not add test data to constants.ts because these will get set as real values in the actual catalog.

Reply:
Moved to mockServices